core/src/com/unciv/logic/city/City.kt

```diff
@@ -403,9 +403,14 @@ class City : IsPartOfGameInfoSerialization {
 
     fun canPlaceNewUnit(construction: BaseUnit): Boolean {
         val tile = getCenterTile()
+        val airUnitCapacity = tile.getCity()?.civ!!.gameInfo.ruleset.modOptions.constants.airUnitCapacity
```

Comment:
No need to go through the tile, we're already in the city

Reply:
Also we don't need to get the uniques for every unit, only for air units, so for performance reasons this should be in the 'when'

Reply:
You are right 😀